src/components/error/FrontEnd/FrontEndError.css

```diff
@@ -0,0 +1,59 @@
+.dash-fe-errors {
```

Comment:
@rmarren1 Could you explain the convention currently used? Does `_dash-*` differ in signification from `dash-*`? My first thought was that the `_` prefix was for donating a private class but `and make it known to users that we are polluting the _dash prefixed namespace` makes me think otherwise.